#! /bin/sh -e

# DP: Do not build gpidump due to PR optimization/9279.

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        ;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

--- gcc/p/utils/Makefile~	2006-06-10 23:11:43.995565704 +0000
+++ gcc/p/utils/Makefile	2006-07-04 08:49:03.733107048 +0000
@@ -41,9 +41,9 @@
 
 # Internal variables
 PFLAGS1=--executable-path=. --unit-path=$(GCC_DIR)/p/rts --unit-path=$(GCC_DIR)/p/units
-EXE=binobj$(exeext) gpidump$(exeext)
-EXE2=binobj gpidump
-DOC=binobj.1 gpidump.1
+EXE=binobj$(exeext)
+EXE2=binobj
+DOC=binobj.1
 PC_WITH_FLAGS=$(PC) --automake --executable-file-name $(UTILS_WARN) $(CFLAGS) $(PFLAGS) $(PFLAGS1)
 
 all: $(EXE) $(DOC)
